openhexa/sdk/pipelines/pipeline.py

```diff
@@ -21,23 +21,28 @@
 from .utils import get_local_workspace_config
 
 logger = getLogger(__name__)
+PIPELINE_MAX_TIMEOUT = 43200
```

Comment:
I don't think this should be hardcoded here. The max should be handled by the backend.